src/components/PeriodSelector/modules/FixedPeriodsGenerator.js

```diff
@@ -251,15 +286,19 @@ function SixMonthlyPeriodType(monthNames, fnFilter) {
         let period = {}
         period.startDate = `${year}-01-01`
         period.endDate = `${year}-06-30`
-        period.name = `${monthNames[0]} - ${monthNames[5]} ${year}`
+        const monthStart = getMonthName(0)
+        const monthEnd = getMonthName(5)
+        period.name = () => `${monthStart} - ${monthEnd} ${year}`
         period.iso = `${year}S1`
         period.id = period.iso
         periods.push(period)
 
         period = {}
         period.startDate = `${year}-07-01`
         period.endDate = `${year}-12-31`
-        period.name = `${monthNames[6]} - ${monthNames[11]} ${year}`
+        const monthStart2 = getMonthName(6)
```

Comment:
`monthStart2`? 🤷‍♂️

Reply:
I was able to remove these constants. So `monthStart2` is gone (yes, I was getting lazy, and tired of this code 🙄 )